app/views/discovered_hosts/_discovered_host_modal.html.erb

```diff
@@ -17,6 +17,8 @@
           <% if show_location_tab? %>
             <%= select_f f, :location_id, Location.my_locations, :id, :to_label, {}, {:size => 'col-md-10'} %>
           <% end %>
+
+          <div>Changing Subnet is not possible for via Customize Host form, use Create Host or Auto provisioning to set Subnet from a Hostgroup.</div>
```

Comment:
`Changing the Host's Subnet is not possible via Customize Host form, use Create Host or Auto provisioning and set the desired Subnet at the Hostgroup.`?